.travis.yml

```diff
@@ -5,8 +5,6 @@ services:
   - mongodb
 before_script:
   - npm install -g gulp
-  - gulp &
-  - sleep 5
```

Comment:
In another PR (I think kyungah's?), this was set to 10. You can just sync and will get that reflected when she merges hers. :)